README.md

```diff
@@ -107,6 +107,24 @@ decorate(Todo, {
 })
 ```
 
+For applying multiple decorators on a single property, you can pass an array of decorators.
```

Comment:
Please mension what the application order is. I assume serializable is applied first, then persist and finally observable?

Reply:
correct